.github/workflows/license_check.yml

```diff
@@ -0,0 +1,56 @@
+name: Check licenses
+
+on: 
+  workflow_dispatch:
```

Comment:
issue: Without `workflow_call` it is not possible to use this workflow in other workflows.

Reply:
fixed